lib/md007.js

```diff
@@ -13,12 +13,14 @@ module.exports = {
   "function": function MD007(params, onError) {
     const indent = Number(params.config.indent || 2);
     const startIndented = !!params.config.start_indented;
+    // start_indent (number; ignored if start_indented is false, else defaults to indent)
+    const startIndent = startIndented ? Number(params.config.start_indent || indent) : 0;
     flattenedLists().forEach((list) => {
       if (list.unordered && list.parentsUnordered) {
         list.items.forEach((item) => {
           const { lineNumber, line } = item;
           const expectedNesting = list.nesting + (startIndented ? 1 : 0);
-          const expectedIndent = expectedNesting * indent;
+          const expectedIndent = startIndent + expectedNesting * indent;
```

Comment:
I think this is the problem - it's double-counting the first indent. I think you want something like:

    let expectedIndent = 0;
    if (expectedNesting >= 1) { expctedIndent += startIndent; }
    if (expectedNesting >= 2) { expectedIndent += (expectedNesting - 1) * indent;

Reply:
good work